ocp_resources/machine_config_pool.py

```diff
@@ -2,8 +2,74 @@
 
 
 class MachineConfigPool(Resource):
+    """
+    MachineConfigPool object. API reference:
+    https://docs.openshift.com/container-platform/4.12/rest_api/machine_apis/machineconfigpool-machineconfiguration-openshift-io-v1.html
+
+    Args:
+        machine_config_selector (object): Matching labels/expressions, to determine which MachineConfig objects
+            to apply this MachineConfigPool object.
+        configuration (dict): Targeted MachineConfig object for the machine config pool, in the following format:
+            {"name": (str), "source": <List of dicts, each representing a MachineConfig resource>}
+        node_selector_labels (object): Matching labels, to determine to which nodes to apply this
+            MachineConfigPool object.
+        node_selector_expressions (object): Matching expressions, to determine to which nodes to apply this
+            MachineConfigPool object.
+        max_unavailable (int/str): Number/percentage of nodes that can go Unavailable during an update.
+        paused (bool): Whether changes to this MachineConfigPool should be stopped.
```

Comment:
`should be paused.`

Reply:
The description of this member in the API says "should be stopped". Do you still want me to change it?

Reply:
leave it as the API say